apps/coscrad-frontend-e2e/src/e2e/resources/digital-texts/admin/digital-text.add-page.e2e.cy.ts

```diff
@@ -0,0 +1,164 @@
+import { AggregateType, ResourceType } from '@coscrad/api-interfaces';
+import { buildDummyAggregateCompositeIdentifier } from '../../../../support/utilities';
+
+const compositeIdentifierForDigitalTextWithoutAPage = buildDummyAggregateCompositeIdentifier(
+    AggregateType.digitalText,
+    1
+);
+
+const compositeIdentifierForDigitalTextThatAlreadyHasAPage = buildDummyAggregateCompositeIdentifier(
+    AggregateType.digitalText,
+    2
+);
+
+const buildDetailRoute = (digitalTextId: string) => `/Resources/DigitalTexts/${digitalTextId}`;
+
+describe(`digital text detail- admin- add page`, () => {
```

Comment:
We should update this test and implementation so that you can only add a page in `icon` view as per Geoff's feedback at standup.